File: testdata/bench/c/BenchmarkTest00011.c
#include <sqlite3.h>

void handle_request(const char *user_input) {
    sqlite3 *db;
    sqlite3_stmt *stmt;
    sqlite3_open(":memory:", &db);
    sqlite3_prepare_v2(db, "SELECT * FROM users WHERE id = ?", -1, &stmt, NULL);
    sqlite3_bind_text(stmt, 1, user_input, -1, SQLITE_TRANSIENT);
    sqlite3_step(stmt);
    sqlite3_finalize(stmt);
    sqlite3_close(db);
}

File: testdata/bench/swift/BenchmarkTest00011.swift
import Vapor
import SQLite3

func handler011(_ req: Request) throws -> String {
    let userId = try req.parameters.require("id")
    var db: OpaquePointer?
    var stmt: OpaquePointer?
    sqlite3_open(":memory:", &db)
    sqlite3_prepare_v2(db, "SELECT * FROM users WHERE id = ?", -1, &stmt, nil)
    sqlite3_bind_text(stmt, 1, userId, -1, nil)
    return "ok"
}
